Percentiles.ipynb

```diff
@@ -200,11 +200,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## The Central Limit Theorem\n",
+    "## The Law of Large Numbers\n",
     "\n",
     "Before we continue with our story, let's review some stats.\n",
     "\n",
-    "You may recall from school that there's a handy thing called the [Central Limit Theorem](https://en.wikipedia.org/wiki/Central_limit_theorem). The CLT basically says this: The average of a large enough sample will be pretty close to the \"real\" average, give or take some error.\n",
+    "You may recall from school that there's a handy thing called the [Law of large numbers](https://en.wikipedia.org/wiki/Law_of_large_numbers). The LLN basically says this: The average of a large enough sample will be pretty close to the \"real\" average, give or take some error.\n",
     "\n",
     "This is pretty handy for measuring a lot of things—it means that even a relatively small sample can give us a pretty good measure of the population average.\n",
```

Comment:
I'd like to see this statement quantified. I don't think the LLN makes guarantees about sample sizes out of the box, but is a general statement on expectations.